github.py

```diff
@@ -7,6 +7,9 @@
 import os
 
 import requests
+from country_list import countries_for_language
```

Comment:
@kdobolyi, can you please follow the PEP 8 guidelines for import order? https://www.python.org/dev/peps/pep-0008/#imports . Please let me know if you have questions; I'm glad to help. I'm sorry for not making it clear that this import ordering is preferred before you started work on this PR. Nice work on this feature and PR!

Reply:
No, this is great -- I normally clean these up in a refactoring step weeks into the project. LMK if they're still in the wrong order (hopefully not).